hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestKeyManagerImpl.java

```diff
@@ -1238,123 +1260,113 @@ public void testListStatus() throws IOException {
   @Test
   public void testRefreshPipeline() throws Exception {
 
-    MiniOzoneCluster cluster = MiniOzoneCluster.newBuilder(conf).build();
-    try {
-      cluster.waitForClusterToBeReady();
-      OzoneManager ozoneManager = cluster.getOzoneManager();
+    OzoneManager ozoneManager = om;
```

Comment:
This function has only really been changed to remove the cluster, (which has been replaced by the TestOmManager's OzoneManager.)  Removing the try block made it look like there were many other changes but they are just whitespace.